website/docs/r/catalog_vapp_template.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_catalog_vapp_template"
+sidebar_current: "docs-vcd-resource-catalog-vapp_template"
+description: |-
+  Provides a VMware Cloud Director vApp Template resource. This can be used to upload and delete OVA files inside a catalog.
+---
+
+# vcd\_catalog\_vapp\_template
+
+Provides a VMware Cloud Director vApp Template resource. This can be used to upload OVA to catalog and delete it.
```

Comment:
Let's add a blue note that this resource replaces the catalog item?

Also, we need a deprecation warning message + a link to use this new resource in the catalog item doc page, don't we?

Reply:
At the moment, we are not replacing the catalog item, but only its usage within the VM.
As part of the subscribed catalog PR (#916), I will write a guide that will include the relationship between catalog items, vApp templates, and media items.